src/pages/ExamsPage/hooks.js

```diff
@@ -3,7 +3,7 @@ import { useSelector, useDispatch } from 'react-redux';
 
 import { RequestKeys } from 'data/constants';
 
-import * as reduxHooks from 'data/redux/hooks/requests';
+import * as reduxHooks from 'data/redux/hooks';
```

Comment:
The contents of `/requests` in already exported from an index file for that module. Got tripped up in my tests by that inconsistency so this is just a bit of clean up.